src/main/kotlin/org/rust/cargo/project/configurable/RsExternalLinterConfigurable.kt

```diff
@@ -5,39 +5,82 @@
 
 package org.rust.cargo.project.configurable
 
+import com.intellij.execution.configuration.EnvironmentVariablesComponent
 import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.ComboBox
 import com.intellij.openapi.ui.DialogPanel
 import com.intellij.ui.EnumComboBoxModel
+import com.intellij.ui.components.Label
 import com.intellij.ui.dsl.builder.*
 import org.rust.RsBundle
 import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.settings.ExternalLinterProjectSettingsService
+import org.rust.cargo.project.settings.externalLinter
+import org.rust.cargo.project.settings.rustSettings
 import org.rust.cargo.toolchain.ExternalLinter
+import org.rust.cargo.toolchain.RustChannel
 import org.rust.cargo.util.CargoCommandCompletionProvider
 import org.rust.cargo.util.RsCommandLineEditor
 import org.rust.openapiext.fullWidthCell
+import javax.swing.JLabel
 
 class RsExternalLinterConfigurable(project: Project) : RsConfigurableBase(project, RsBundle.message("settings.rust.external.linters.name")) {
+    private val settings: ExternalLinterProjectSettingsService = project.rustSettings.externalLinter
+
+    private val additionalArguments: RsCommandLineEditor =
+        RsCommandLineEditor(project, CargoCommandCompletionProvider(project.cargoProjects, "check ") { null })
```

Comment:
I'm not sure but, probably, because of completion support here the background of the text field differs from the same background in `Rustfmt` settings

<img width="734" alt="Screenshot 2022-08-22 at 12 38 49" src="https://user-images.githubusercontent.com/2539310/185902835-d5acaaa3-b36e-46d6-9cee-c1b16ac4bee6.png">
<img width="733" alt="Screenshot 2022-08-22 at 12 38 58" src="https://user-images.githubusercontent.com/2539310/185902893-3491889c-a2d4-49bc-8446-97b1f227fe56.png">

Reply:
I see, but this bug(?) is not relevant to this PR, the stable plugin has it as well. Do you want me to fix it in this PR?